### PR TITLE
Changed Nix installer to DeterminateSystems to avoid enabling flakes

### PR DESCRIPTION
**Pull request by star-gazer111**
Fixes [4777](https://github.com/fedimint/fedimint/issues/4777)

### CODE REVIEW

**Comment by star-gazer111** [comment]
@dpc I have made changes made as u suggested and I had to open up a new PR because I accidentally deleted the fork in which the original PR was opened. Please have a look and confirm. The above command works perfectky and so `nix develop` works perfectly as well.

**Review by elsirion — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -91,4 +91,4 @@ You can then interact with the guardians and lightning gateway through the mproc
 If you want to run with UIs, see the [UI](https://github.com/fedimint/ui) repo for developer environment instructions.
 
 # Contributing
-For contribution guidelines, Areas of contributions and how to get involved, please refer to the [Contributing Guidelines](CONTRIBUTING.md).
\ No newline at end of file
+For contribution guidelines, Areas of contributions and how to get involved, please refer to the [Contributing Guidelines](CONTRIBUTING.md).
```
Comment (maan2003) [inline README.md:94]: wait what changed here

Comment (star-gazer111) [reply README.md:94]: I didn't change anything here.

Comment (maan2003) [reply README.md:94]: looks like new line missing on left, so it is fine

**Review by maan2003 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (star-gazer111) [reply README.md:94]: Ah got it. 
I think we can merge this now.

Comment (dpc) [reply README.md:94]: You're not trying to sneak-in some ssh backdoors in whitespace changes, are you? ;)

Comment (star-gazer111) [reply README.md:94]: Obviously not ser ;)

**Pull request merged**